ci/tiledb-java-fat-jar.yml

```diff
@@ -0,0 +1,72 @@
+steps:
+  - download: current
+    patterns: '**/*.tar.gz'
+
+  - bash: |
+      unset SYSTEM
+      set +e
+      mv ../*.tar.gz/*.tar.gz .
+
+      mkdir -p ./build/install/lib
+      mkdir ./build/install/lib64
+      mkdir ./build/tiledb_jni/
+      mkdir ./build/tiledb_jni/Release
+      mkdir ./build/install/bin
+
+      for arch in $(ls | grep .tar.gz)
+      do
+        tar -xf $arch
+      done
+
+      # OSX
+      mv libtiledb.dylib ./build/install/lib
+      mv libtiledbjni.dylib ./build/tiledb_jni
+
+      # Linux
+      cp libtiledb.so ./build/install/lib
+      mv libtiledb.so ./build/install/lib64
+      mv libtiledbjni.so ./build/tiledb_jni
+
+      # Windows
+      mv tbb.dll ./build/install/bin
+      mv tiledb.dll ./build/install/bin
+      mv tiledbjni.dll ./build/tiledb_jni/Release
+
+      ./gradlew assemble
+
+      PROJECT_VERSION=$(./gradlew properties -q | grep "version:" | awk '{print $2}')
+
+      cp build/libs/*.jar $BUILD_BINARIESDIRECTORY
+
+    displayName: 'Building the Fat Jar'
+
+  - bash: |
+      set -e pipefail
+      # Display log files if the build failed
+      echo "Dumping log files for failed build"
+      echo "----------------------------------"
+      for f in $(find $BUILD_REPOSITORY_LOCALPATH -name *.log);
+        do echo "------"
+           echo $f
+           echo "======"
+           cat $f
+        done;
+    condition: failed() # only run this job if the build step failed
+    displayName: "Print log files (failed build only)"
+
+  - task: ArchiveFiles@2
+    inputs:
+      rootFolderOrFile: '$(Build.BinariesDirectory)'
+      includeRootFolder: false
+      archiveType: 'tar' # Options: zip, 7z, tar, wim
+      tarCompression: 'gz' # Optional. Options: gz, bz2, xz, none
+      archiveFile: $(Build.ArtifactStagingDirectory)/tiledb-java-release-$(Build.SourceBranchName)-$(commitHash).tar.gz
+      replaceExistingArchive: true
+      verbose: true # Optional
+    condition: succeeded()
+
+  - task: PublishBuildArtifacts@1
+    inputs:
+      pathtoPublish: $(Build.BinariesDirectory)
+      artifactName: fat_jar
```

Comment:
Change the artifact name.

Reply:
Got it. These three are the only files `$(Build.BinariesDirectory)`. Can you check this line? https://github.com/TileDB-Inc/TileDB-Java/pull/171/files#diff-f6cc1790cd29925bdb06907055c4b05bR39